src/cryptography/hazmat/bindings/openssl/_conditional.py

```diff
@@ -438,4 +480,5 @@ def cryptography_has_verified_chain():
     ),
     "Cryptography_HAS_ENGINE": cryptography_has_engine,
     "Cryptography_HAS_VERIFIED_CHAIN": cryptography_has_verified_chain,
+    "Cryptography_HAS_X509_CB_ISSUER_CHECK": cryptography_has_x509_cb_issuer_check,
```

Comment:
This line needs to be wrapped with parentheses due to pep8.

Reply:
Done